src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -118,6 +118,7 @@ def run_python_test(test_target, pytest, python_setup, source_root_config, subpr
     env=pex_exe_env,
     input_files=merged_input_files,
     description=f'Run Pytest for {test_target.address.reference()}',
+    timeout_seconds=getattr(test_target, 'timeout', 60)
```

Comment:
These timeouts are actually configured by options registered on `TestRunnerTaskMixin`, rather than being hardcoded this way. It's probably fine to hardcode this for now, but referencing that implementation would be good.